tests/test_converters.py

```diff
@@ -292,6 +293,70 @@ def test_converter(self):
                      ('batch', 'channel', 'height', 'width'))
 
 
+class TestCelebA(object):
+    def setUp(self):
+        numpy.random.seed(21 + 1 + 2016)
+        self.tempdir = tempfile.mkdtemp()
+        cwd = os.getcwd()
```

Comment:
Not needed for this PR perhaps, but this reminds me that we finally need the context manager that @hantek wrote in #265.